main/lsp/watchman/WatchmanProcess.cc

```diff
@@ -95,8 +95,12 @@ void WatchmanProcess::start() {
                     // Gracefully handle deserialization errors, since they could be our fault.
                     logger->error("Unable to deserialize Watchman request: {}\nOriginal request:\n{}", e.what(), line);
                 }
+            } else if (d.HasMember("state-enter")) {
+                // Deliberately keep these from "Unknown response" logging.
+            } else if (d.HasMember("state-leave")) {
+                // Deliberately keep these from "Unknown response" logging.
```

Comment:
You may want to describe a little bit about what these are?

Maybe even just a link to the docs or something. Up to you.

Reply:
I added some links and expanded the comments.